<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CardButton</class>
 <widget class="QWidget" name="CardButton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>93</width>
    <height>170</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>93</width>
    <height>170</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QFrame" name="upperSpacerWidget"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="cardLayout">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="lowerSpacerWidget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
